### ISSUE TITLE
Feature request: Show the number of bookmarks in each folder (as number after the folder name, or as a separate column)

### ISSUE THREAD

**Issue by raindropsfromsky** [brian-girko/bookmarks-commander#52]
Power users of the browser want to select a node in the bookmarks tree (i.e., a folder), and open all the bookmarks at once.
However, if the folder has accumulated too many bookmarks, it becomes impossible/impractical to open them all at once.

In such cases, the user may want to find all folders that have a large number of bookmarks and split them in several folders.

For example, suppose I have folders named A-C, D-F, G-J, K-Z.
- If I find that too many bookmarks  have accumulated in D-F, I may split them in new folders named D, E-F.
- If there are too many bookmarks in D folder, I may split them into D1, D2, etc..

After this splitting, it is possible for me to launch all bookmarks in any folder.

To make this possible, we need to know the number of bookmarks in any folder without having to open and inspect the folders one by one.

In other words, the number of bookmarks should be shown against the folder name, like this:

A-C   (10)
D-F    (2)
G-J     (20)
K-Z     (30)

In this example, I will know at a glance that I have to split the G-J and K-Z folders.

This feature can be made optional (let the user turn it off).

Alternatively, a column should be added to the display to show the number of bookmarks in each folder.
(This column will remain blank for bookmark entries)

**Comment by TheEditor69** [comment]
I second this request
